kfs-core/src/main/java/org/kuali/kfs/gl/businessobject/Entry.java

```diff
@@ -875,4 +877,20 @@ public DocumentTypeEBO getReferenceFinancialSystemDocumentTypeCode() {
         return referenceFinancialSystemDocumentTypeCode;
     }
 
+    public Long getEntryId() {
+        return entryId;
+    }
+
+    public void setEntryId(Long entryId) {
+        this.entryId = entryId;
+    }
+
+    public String getGecDocumentNumber() {

```

Comment:
This is used in the GEC Entry lookup, brings snappy responsiveness to a previously called-out pain spot, and serves to block selectability of the entry by subsequent GEC docs. This is also the column in the lookup that will be hyperlinked to the GEC doc that has been previously associated, explicitly called out in the functional spec. As for more work to sync: it's but a single point of intersection, and is actually dead simple logic located in one short method: GeneralErrorCorrectionDocument.doRouteStatusChange().